src/nodejs/lambda-handlers/submission.js

```diff
@@ -147,15 +147,31 @@ async function submitMethod(event, user) {
 }
 
 async function reviewMethod(event, user) {
-  const approvedUserPrivileges = ['ADMIN', 'REQUEST_REVIEW', 'REQUEST_REVIEW_MANAGER'];
+  const approvedUserPrivileges = ['ADMIN', 'REQUEST_REVIEW', 'REQUEST_REVIEW_MANAGER', 'CREATE_STEPREVIEW', 'REMOVE_STEPREVIEW'];
   const { id, approve } = event;
+  const userId = user.id;
   if (user.user_privileges.some((privilege) => approvedUserPrivileges.includes(privilege))) {
     const status = await db.submission.getState({ id });
     const stepType = status.step.type;
     let eventType;
+    let param;
     if (approve === 'false' || !approve) {
```

Comment:
Handle the db queries in this if else block so that they don't throw errors in the event of an approval or rejection by a user that is not in the table for a given step or the approval or rejection of a step that is not in the table.